solvesCryptarithm/README.md

```diff
@@ -0,0 +1,65 @@
+# Python Autograder Template with solvesCryptarithm
+
+### Assessment Language
+
+Python
+
+### Autograder Language
+
+Python
+
+### Autograding Environment Packages
+
+The python3 environment package is not installed on the default autograding_image. To run this autograder, run the following commands in Terminal after adding python3 to the `apt-get install` in your Dockerfile (~/Tango/vmms/Dockerfile), like in the example [here](Dockerfile).
+```
+cd ~/Tango
+docker build -t autograding_image vmms/
```

Comment:
since you added a dockerfile here, you can just instruct them to run `docker build -t autograding_image .` inside the solvescryptarithm directory